app/models/concerns/hyrax/file_set/belongs_to_works.rb

```diff
@@ -4,7 +4,7 @@ module BelongsToWorks
       extend ActiveSupport::Concern
 
       included do
-        before_destroy :remove_representative_relationship
+        before_destroy :remove_representative_relationship, :remove_rendering_relationship
```

Comment:
I wonder if the three changes you made to this module ought to move to the new `Hyrax::HasRendering` module so that all the rendering stuff is together.

Reply:
Can we please not add callbacks?  Can you put this in the actor stack instead?  This will make it much easer to integrate with Valkyrie.

Reply:
I've moved this into the vile_set_actor, and also removed remove_representative_relationship as this is also in the (unlink_works) file_set_actor